apps/cms/package.json

```diff
@@ -28,6 +28,7 @@
     "graphql": "^16.6.0",
     "graphql-tag": "^2.12.6",
     "next": "13.2",
+    "next-urql": "^5.0.0",
```

Comment:
I dont think we need this, we don't have SSR - you can use GraphqlProvider with useQuery or raw createClient.

Reply:
I have the same impression

Reply:
I was suggesting how it's done in search app, it also uses next-urql. I'll remove it

Reply:
Just ensure its needed, @krzysztofwolski ?

Reply:
- Might be a leftover library from the hackathon.
- Not sure if `toPromise` method for queries is buildin or added by this package

The fastest way to ensure will be to remove it from codegen config and checking if the project builds.

Reply:
I don't see it in codegen config (I understand it as `.graphqlrc.yml`), but after removing it from project, project itself builds. So it's fine I guess.

Reply:
Yup, then it can be removed